addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -288,6 +297,25 @@ export class Rtc extends Record {
             }
             this.call();
         }, 30_000);
+
+        this.multiTab.subscribe("discuss.rtc/recover", async ({ detail }) => {
+            if ((new Date().getTime() - detail.timestamp) > RECOVERY_DELAY) {
+                return;
+            }
+            // Delayed for the new main tab election
```

Comment:
This really looks fragile:
- No main => no guarantee that the election will occur in the given 100ms.
- Main => no guarantee that the election won't start in 90ms => we still are in a middle of the election.
We should provide a way to wait for the election to be done instead.